sljit_src/sljitNativeX86_common.c

```diff
@@ -557,7 +557,7 @@ SLJIT_API_FUNC_ATTRIBUTE void* sljit_generate_code(struct sljit_compiler *compil
 					SLJIT_ASSERT(put_label->label);
 					put_label->addr = (sljit_uw)code_ptr;
 #if (defined SLJIT_CONFIG_X86_64 && SLJIT_CONFIG_X86_64)
-					code_ptr = generate_put_label_code(put_label, code_ptr, (sljit_uw)(SLJIT_ADD_EXEC_OFFSET(code, executable_offset) + put_label->label->size));
+					code_ptr = generate_put_label_code(put_label, code_ptr, (sljit_uw)(SLJIT_ADD_EXEC_OFFSET(code, executable_offset)) + put_label->label->size);
```

Comment:
wouldn't the fact that we are trying to use an out of scope address for a label be a reason to trigger a compilation error instead?

Reply:
I can imagine someone creating a label by hand to point somewhere in memory (ex: some IO map) and therefore hitting this codepath if they made a mistake, so still think failing to compile will be helpful here and definitely preferably to writing somewhere else in memory and hopefully crashing soon after